src/Symfony/Component/Form/Extension/Core/ChoiceList/ChoiceList.php

```diff
@@ -100,7 +108,7 @@ public function __construct($choices, array $labels, array $preferredChoices = a
      * @param array              $labels           The labels belonging to the choices.
      * @param array              $preferredChoices The choices to display with priority.
      */
-    protected function initialize($choices, array $labels, array $preferredChoices)
+    protected function initialize($choices, $labels, $preferredChoices)
```

Comment:
Can you add a note in the UPGRADE file, it looks like this is the only remaining thing to do before merging.